Khasi/Khasi_song_lyrics

```diff
@@ -0,0 +1,33 @@
+Hoi Kiw
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.